docs/content/en/docs-dev/user-guide/managing-application/deployment-chain.md

```diff
@@ -6,7 +6,7 @@ description: >
   Specific guide for configuring chain of deployments.
 ---
 
-For users who want to use PipeCD to build a complex deployment flow, which contains multiple applications across multiple application kinds and roll out them to multiple clusters gradually or promoting across environments, this guideline will show you how to use PipeCD to archive that requirement.
```

Comment:
Thank you :) There might be the same typo in the other version's docs.
Could you check them?

Reply:
@ffjlabo Thanks, I fixed v0.39.x - v0.46.x too!